uast/uast-kotlin/test/AbstractKotlinUastTest.kt

```diff
@@ -95,7 +98,8 @@ abstract class AbstractKotlinUastTest : AbstractUastTest() {
         area.getExtensionPoint(UastLanguagePlugin.extensionPointName).registerExtension(KotlinUastLanguagePlugin(), project)
         area.getExtensionPoint(UEvaluatorExtension.EXTENSION_POINT_NAME).registerExtension(KotlinEvaluatorExtension(), project)
 
-        project.registerService(
+        val application = ApplicationManager.getApplication() as MockComponentManager
```

Comment:
This trick deserves a comment (since it took me about an hour to figure this out!).